src/ert/ensemble_evaluator/_builder/_legacy.py

```diff
@@ -214,6 +215,7 @@ async def _evaluate_inner(  # pylint: disable=too-many-branches
                     ee_token=self._config.token,
                     on_timeout=on_timeout,
                 )
+                scheduler_logger.debug("Experiment ran on ORCHESTRATOR: job_queue")
```

Comment:
I concur with Håvard, and I also think this should be JobQueue as that's what we've been calling it.

Reply:
Then we need to change to big-poly workflow, which is fine I guess. Will do.

Reply:
Just keep the strings as they were prior to this PR. Not worth the effort to change it.